sorc/build_ww3prepost.sh

```diff
@@ -85,7 +86,7 @@ if (( rc != 0 )); then
   echo "Fatal error in cmake."
   exit "${rc}"
 fi
-make -j 8
+make -j "${BUILD_JOBS:-8}"
```

Comment:
@DavidHuber-NOAA Okay I see this in the code now.   I'm not sure why there are issues...

Reply:
@JessicaMeixner-NOAA I believe I found the issue.  I had killed the ww3prepost build then restarted it.  However, the build_ww3prepost.sh script does not delete the build directory.  I've added in a `[[ -d ${path_build} ]] && rm -rf ${path_build}` which solves the issue.  Typically, the ww3prepost job cleans out the build directory at the end of the build so this probably hasn't been an issue.

Reply:
I'm glad it was a simple thing, I was struggling to think of another cause.   Thanks for these updates! It'll be exciting to have a faster build!